apps/services/auth-public-api/src/app/modules/delegations/meDelegations.controller.ts

```diff
@@ -123,28 +109,28 @@ export class MeDelegationsController {
   @Scopes(AuthScope.readDelegations)
   @FeatureFlag(Features.customDelegations)
   @Get(':delegationId')
-  @ApiOperation({
+  @Documentation({
+    response: { status: 200, type: DelegationDTO },
     description: `Finds a single delegation by ID where the authenticated user is either giving or receiving.
        Does not include delegations from NationalRegistry or CompanyRegistry.`,
+    params: [
+      {
+        name: 'delegationId',
+        type: 'string',
+        description: 'Delegation ID.',
+      },
+    ],
+    queries: [
```

Comment:
The name `queries` confused me, since I associate the plural form with search/database queries rather than the query string.

Would `query` be ok? Note that in Express, there's `request.params` and `request.query`.